hi-ml/src/health_ml/runner.py

```diff
@@ -297,7 +299,7 @@ def after_submission_hook(azure_run: Run) -> None:
             azure_run_info = submit_to_azure_if_needed(
                 input_datasets=input_datasets,  # type: ignore
                 submit_to_azureml=False,
-                environment_variables=environment_variables,
+                environment_variables=self.additional_environment_variables(),
```

Comment:
[Minor] Does `additional_environment_variables()` need to be called down here inside each `if/else` case instead of once above (`environment_variables = ...`)?

Reply:
good point, we can call it once above 
but note that this call is only reached once anyways, we're either running outside or in aml